README.md

```diff
@@ -76,6 +76,19 @@ MIDDLEWARE = [
 ]
 ```
 
+## 2FA with OTP
+
+  You can require users to enter an OTP (One Time Password) along with their email to get the magic link.
+ - What is a OTP ? It is a short code (6 to 8 digits), usually generated by a dedicated app on a user's device (smartphone, hardware). See https://en.wikipedia.org/wiki/Time-based_One-time_Password_algorithm
+ - We use the [django-otp](https://django-otp-official.readthedocs.io/en/stable/) package. You must have it installed and configured in your project
+ - How to enable ?
+   1. install `django_otp` and run the migrations in your project (prerequisite)
+   2. Set `MAGICAUTH_ENABLE_2FA = True` in your settings
+ - Usage ?
+   - An OTP input will show up in the login page, below the Email. If you have a custom login template, don't forget to add `OTP_form`. See templates/magicauth/login.html` for an example that works with TOTPs
+   - If the OTP is not valid, the form returns an error and the magic link is not sent
```

Comment:
For testing purposes, you can generate OTPs for a given user from the command line.
Add `'django_otp.plugins.otp_static'` to INSTALLED_APPS.
Then you can use [addstatictoken](https://django-otp-official.readthedocs.io/en/stable/overview.html#addstatictoken) from command line. E.g : 
`python manage.py addstatictoken -t 123456 "thierry@coucou.fr"`